src/applications/edu-benefits/10203/containers/StemEligibilityView.jsx

```diff
@@ -0,0 +1,289 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+import { isChapter33 } from '../helpers';
+import captureEvents from '../analytics-functions';
+
+export class StemEligibilityView extends React.Component {
+  onChange = property => {
+    this.props.onChange({
+      ...this.props.formData,
+      ...property,
+    });
+  };
+
+  inReviewEditMode = () => this.props.onReviewPage && this.props.reviewMode;
+  showNotApplyingToStemInformation = () =>
+    !this.props.onReviewPage &&
+    this.props.determineEligibility !== undefined &&
+    !this.props.determineEligibility;
+
+  iconClass = indication =>
+    classNames('fa', {
+      'fa-check': indication,
+      'vads-u-color--green': indication,
+      'fa-times': !indication,
+      'vads-u-color--gray-medium': !indication,
+    });
+
+  iconText = indication =>
+    indication ? `You answered yes to` : `You answered no to`;
+
+  renderCheck = (classes, title, text) => (
+    <li className="vads-u-margin-bottom--0">
+      <span className="fa-li">
+        <i className={classes} aria-hidden="true" />
```

Comment:
Will you provide some screenshots?  There is a lot of code in this PR which makes it difficult to see if this icon is being used semantically or not.

Reply:
Sure, here's a screenshot of what this is referring too.
![image](https://user-images.githubusercontent.com/48804654/78595234-acfc0580-7817-11ea-9361-8d4ddce8e092.png)

As you can see, it's the review of your Stem Scholarship eligibility that exists in the 1995 form. Only difference here, is it's saying it's  a `Form 22-10203` which is the potential new name for the STEM Scholarship form.

Reply:
It looks like this icon is being used semantically.  Since that is the case, a `title` should be provided for accessibility reasons.

https://fontawesome.com/v4.7.0/accessibility/

Reply:
I've added in the icon.